res/css/rethemendex.sh

```diff
@@ -5,8 +5,7 @@ cd `dirname $0`
 {
     echo "// autogenerated by rethemendex.sh"
 
-    find . \! \( -path ./themes -prune \) -iname _\*.scss |
-        fgrep -v _components.scss | LC_ALL=C sort |
+    find . -iname _\*.scss | fgrep -v _components.scss | LC_ALL=C sort |
```

Comment:
Explaining the change here in the commit/in a comment would be nice, but this seems to have worked so LGTM.

Reply:
have commented for posterity in the code.